src/compass/utils/h5_helpers.py

```diff
@@ -79,6 +79,33 @@ def add_dataset_and_attrs(group, meta_item):
         val_ds.attrs[key] = _as_np_string_if_needed(val)
 
 
+def _determine_fill_value(dtype):
```

Comment:
I might be wrong, but I'm not sure where this function is used.
The commit I'm seeing right now is `3af5b7eeaa42c45fb4b71385b3da28c5137fb992`

Reply:
It was in 98be0896a034da26990d4dc4243ae2036613cd8b but got merged away. Working to restore.